### PR TITLE
Add list group members API endpoint

### PR DESCRIPTION
**Pull request by lvreynoso**
### Summary:
- **What:** This is a first iteration of a list group members API endpoint. Pulls information from the database for all users in the current user's group. Members sorted alphabetically.
- **Ticket:** [sc187629](https://app.shortcut.com/genepi/story/187629)
- **Env:** [https://listmembers-backend.dev.czgenepi.org/v2/groups/members](https://listmembers-backend.dev.czgenepi.org/v2/groups/members)

### Demos:
Screen Shot 2022-05-17 at 16 17 57

### Notes:
TODO: tests

### Checklist:
- [ ] I merged latest `<base branch>`
- [ ] I manually verified the change
- [ ] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/backend/aspen/api/schemas/usergroup.py [hunk 1]
```diff
@@ -42,3 +34,17 @@ class UserPostRequest(BaseRequest):
     group_admin: Optional[bool] = False
     system_admin: Optional[bool] = False
     auth0_user_id: str
+
+
+class MemberResponse(UserBaseResponse):
+    email: str
+    group_admin: bool
```
Comment (jgadling) [inline src/backend/aspen/api/schemas/usergroup.py:41]: Should we include name here too?

File: src/backend/aspen/api/views/groups.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+from aspen.api.auth import get_auth_user
+from aspen.api.deps import get_db
+from aspen.api.schemas.usergroup import AllGroupsMembersResponse
+from aspen.database.models import User
+
+router = APIRouter()
+
+
+@router.get("/members", response_model=AllGroupsMembersResponse)
```
Comment (jgadling) [inline src/backend/aspen/api/views/groups.py]: I think this route should be `/v2/groups/[group_id]/members`

File: src/backend/aspen/api/schemas/usergroup.py (hunk 1 above)
Comment (lvreynoso) [reply src/backend/aspen/api/schemas/usergroup.py:41]: That's part of the `UserBaseResponse`.

File: src/backend/aspen/api/schemas/usergroup.py [hunk 3]
```diff
@@ -42,3 +34,17 @@ class UserPostRequest(BaseRequest):
...
+
+
+class GroupMembersResponse(BaseResponse):
+    group: GroupResponse
```
Comment (jgadling) [inline src/backend/aspen/api/schemas/usergroup.py]: Can we have the group members endpoint just return all group members, and include group details in a separate `/v2/groups/[group_id]` endpoint?

**Review by jgadling — CHANGES_REQUESTED** [review]

**Review by jgadling — APPROVED** [review]

**Pull request merged**